cobalt/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -113,9 +114,9 @@ public StarboardBridge(
 
     Log.i(TAG, "StarboardBridge init.");
 
-    // Make sure the JNI stack is properly initialized first as there is
+    // Make sure the JNI stack is properly initialized first as there is a
     // race condition as soon as any of the following objects creates a new thread.
-    nativeInitialize();
+    initJNI();
```

Comment:
When CoAT stops, JNI will be detached from the jvm thread. 
In lts.25, the entry point is the Destroy() method in jni_env_ext.cc, do you still have the method called when the app stops?

Reply:
Damn this is a good fine. How did we have a JNI shutdown in SbThread orz